src/backend/joanie/core/models/certifications.py

```diff
@@ -153,15 +159,33 @@ def _set_localized_context(self):
         title_object = (
             self.order.product if self.order else self.enrollment.course_run.course
         )
-
+        new_images = set()
         for language, __ in settings.LANGUAGES:
             context[language] = {
                 "course": {
                     "name": title_object.safe_translation_getter(
                         "title", language_code=language
                     ),
                 },
-                "organizations": [
+                "organizations": [],
+            }
+
+            for organization in self.course.organizations.all():
+                signature_checksum = file_checksum(organization.signature)
```

Comment:
You don't check before if `orgaization.signature` exists ?

Reply:
Yup. It's missing.